index.html

```diff
@@ -0,0 +1,55 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="home.css">
+    <link rel="stylesheet" href="reset.css">
```

Comment:
put reset.css above home.css. so the default styling made by the browser is reset before you add your own styles to the project.